library/oid.c

```diff
@@ -649,6 +650,7 @@ FN_OID_TYPED_FROM_ASN1(oid_md_alg_t, md_alg, oid_md_alg)
 FN_OID_GET_ATTR1(mbedtls_oid_get_md_alg, oid_md_alg_t, md_alg, mbedtls_md_type_t, md_alg)
 FN_OID_GET_OID_BY_ATTR1(mbedtls_oid_get_oid_by_md, oid_md_alg_t, oid_md_alg, mbedtls_md_type_t, md_alg)
 
+#if defined(MBEDTLS_MD_C)
```

Comment:
Is this supposed to be here?

Reply:
I think so. Initially this guard was before `oid_md_alg` but also covering this. It just moved here when I wanted it to stop guarding `oid_md_alg`.

(In the long run, it should probably be removed entirely, but I was just trying to make a minimal amount of changes.)

Reply:
> In the long run, it should probably be removed entirely,

See #6136